server/controller/index.js

```diff
@@ -18,8 +18,21 @@ exports.addQuestion = async (ctx) => {
   return ctx.body;
 };
 
+exports.getAllQuestions = async (ctx) => {
```

Comment:
We don't need two controllers for this, as we should never return every questions without a limit.